configs/cassandra-true.textproto

```diff
@@ -0,0 +1,88 @@
+# Some rules need a workaround because the configuration value is not on the same line of the key.
+# Therefore, a workaround is needed. Using the \0 delimiter does not seem to execute the regex on all file content
+#
+# The approach used in the following rules set the file content split delimiter to "true" and checks with a regexp
+# if the required configuration value is enabled or not. The regexp was thoroughly tested to avoid false positives.
+
+benchmark_configs: {
+  id: "cassandra-auditing"
+  compliance_note: {
+    version: { version: "1.0.0" benchmark_document: "CIS Apache Cassandra" }
+    title: "Ensure that auditing is enabled"
+    description:
+      "Audit logging in Cassandra logs every incoming CQL command request, Authentication "
+      "(successful as well as unsuccessful login) to C* node. Currently, there are two "
+      "implementations provided, the custom logger can be implemented and injected with the "
+      "class name as a parameter in cassandra.yaml."
+    rationale:
+      "Unauthorized attempts to create, drop or alter users or data should be a concern."
+    remediation:
+      "Enable the Cassandra Auditing feature. Note that on the Open Source version this feature is available only starting from version 4."
+    cis_benchmark: {
+      profile_level: 1
+      severity: HIGH
```

Comment:
For now we keep the severity of the CIS findings LOW as they're compliance findings as opposed to RCE vulns. So I'd say let's change these to LOW as well.